collectors/ebpf.plugin/ebpf.c

```diff
@@ -930,13 +371,24 @@ static inline void ebpf_enable_specific_chart(struct  ebpf_module *em, int disab
     em->global_charts = 1;
 }
 
+/**
+ * Enable all charts
+ *
+ * @param apps what is the current status of apps
+ */
 static inline void ebpf_enable_all_charts(int apps) {
     int i ;
     for (i = 0 ; ebpf_modules[i].thread_name ; i++ ) {
         ebpf_enable_specific_chart(&ebpf_modules[i], apps);
     }
 }
 
+/**
+ * Enable the specified chart group
+ *
+ * @param enable         enable (1) or disable (0) chart
+ * @param disable_apps   was the apps disabled?
+ */
 static inline void ebpf_enable_chart(int enable, int disable_apps) {
```

Comment:
The naming of parameters here seems confusing to me. `enable` seems to be the index of eBPF module?

Reply:
Done!